src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./Checkbox";
```

Comment:
Você pode exportar os tipos no index também, assim caso alguém for usar em algum outro componente ele poderá importar os tipos do index